wire-ios-sync-engine/Source/Calling/ZMConversation+VoiceChannel.swift

```diff
@@ -23,7 +23,9 @@ private var voiceChannelAssociatedKey: UInt8 = 0
 public extension ZMConversation {
     /// NOTE: this object is transient, and will be re-created periodically. Do not hold on to this object, hold on to the owning conversation instead.
     var voiceChannel: VoiceChannel? {
-            guard conversationType == .oneOnOne || conversationType == .group else { return nil }
+        guard conversationType == .oneOnOne || conversationType == .group else {
+            return nil
+        }
 
             if let voiceChannel = objc_getAssociatedObject(self, &voiceChannelAssociatedKey) as? VoiceChannel {
```

Comment:
nitpick: somehow the indentation looks not fitting..

Reply:
in order not to wait for the CI, I'll pass the nitpick @markusfassbender